.azure_pipelines_snp.yml

```diff
@@ -33,28 +33,47 @@ jobs:
       vmImage: ubuntu-20.04
     steps:
       - script: |
```

Comment:
More of a cleanup job that can probably wait a further PR: we could split this single step into multiple for easier debugging/nicer output in Azure DevOps.

Reply:
Great idea
https://github.com/microsoft/CCF/pull/4662/checks?check_run_id=9792088968